docs/physics/tardisgamma/packetinitialization.rst

```diff
@@ -0,0 +1,28 @@
+Packet Initialization
+=====================
+
+The gamma ray portion of tardis also uses the idea of packets of photons from :cite:`AbbottLucy85` and :cite:`Lucy99` (see :doc:`initialization`)
+These packets are given an energy equal to the total energy divided by the number of packets and a frequency that is equal to the packet energy divided by Planck's constant, h.
+
+The packets are also given a direction made up of two angles, :math:`\theta` and :math:`\phi`, where  :math:`\theta` is a polar angle between 0 and :math:`\pi` and :math:`\phi` is an azimuth angle between 0 and :math:`2\pi`.
+We sample these angles using the following equations :cite:`CarterCashwell75`:
+
+:math:`\cos{\theta}` = 1-2 z \ :sub:`1`\
```

Comment:
Please check if the equations can be placed centrally

Reply:
The equations are now placed centrally.